src/components/ae-list-item/ae-list-item.vue

```diff
@@ -1,57 +1,89 @@
 <template>
-    <li class="ae-list-item" :class="{ [fill]: Boolean(fill) }">
-      <slot />
-    </li>
+  <component
+    :is="renderAs"
+    :to="to"
+    :class="[fill, inactive && 'inactive']"
+    class="ae-list-item"
+    @click="$emit('click', $event)"
+  >
+    <!-- @slot List item content -->
+    <slot />
+    <div class="space" />
```

Comment:
@davidyuk 

In here why are you inserting a `space` div, what is its specific functionality?

Reply:
In general, ListItem should be able to render elements on the left and the right sides. The same can be implemented using `justify-content: space-between` but then slots should be wrapped with divs because if we are passing several elements into the slot, they will be aligned along free space. `<div class="space" />` has a poor semantic, but I don't think that two wrapping divs are better.
Usage example: https://github.com/aeternity/aepp-identity/blob/be5f562a43e39545aafce46909db089c98c23ee5/src/pages/Settings.vue#L15-L21